server/@types/PlanType.ts

```diff
@@ -28,4 +28,5 @@ export type PlanType = {
   updatedBy: string
   agreementDate: string
   goals: Goal[]
+  readOnly: boolean
```

Comment:
What's the need for this? Isn't the `read-only` attribute determined by the `READ_ONLY` role coming from OASys  (maintained in the handover context)

Reply:
removed in https://github.com/ministryofjustice/hmpps-sentence-plan-ui/pull/206/commits/057bb2e02687440e511407401ee7829f85c2667b